src/auth/baseexternalclient.ts

```diff
@@ -53,12 +53,13 @@ export const EXPIRATION_TIME_OFFSET = 5 * 60 * 1000;
  * 3. external_Account => non-GCP service (eg. AWS, Azure, K8s)
  */
 export const EXTERNAL_ACCOUNT_TYPE = 'external_account';
-/** Cloud resource manager URL used to retrieve project information. */
+/**
+ * Cloud resource manager URL used to retrieve project information.
+ *
+ * @deprecated use {@link BaseExternalAccountClient.cloudResourceManagerURL} instead
+ **/
 export const CLOUD_RESOURCE_MANAGER =
   'https://cloudresourcemanager.googleapis.com/v1/projects/';
-/** The workforce audience pattern. */
-const WORKFORCE_AUDIENCE_PATTERN =
```

Comment:
Please revert these changes -- the audience is always iam.googleapis.com.

Reply:
Revert PR: https://github.com/googleapis/google-auth-library-nodejs/pull/1740